modules/odc-creating-apiserversource.adoc

```diff
@@ -31,6 +33,11 @@ image::apiserver-odc-config-1.png[Select the Service Account Name and Sink]
 +
 image::toplogy-odc-apiserver.png[ApiServerSource Topology view]
 
+[NOTE]
+====
+If a **URI sink** is used, the URI can be modified by right clicking on the **URI sink** -> `Edit URI`.
```

Comment:
Oops, just noticed one thing, should `Edit URI` also be **Edit URI** ?

Reply:
Also, not sure **URI sink** needs to be bold in the first instance? Not sure though, your call.

Reply:
good catch, fixed, thanks